jsonschema/Cargo.toml

```diff
@@ -17,19 +17,16 @@ name = "jsonschema"
 
 [features]
 cli = ["structopt"]
-default = ["reqwest", "cli"]
+default = ["resolve-http", "resolve-file", "cli"]
 draft201909 = []
 draft202012 = []
-reqwest-native-tls = ["reqwest/native-tls"]
```

Comment:
As it was asked in #306 I think [this](https://github.com/ahungrynoob/jsonschema/blob/master/Cargo.toml) is the only place where it was relevant. So, with `default-features = false`, these changes should not cause any issues there

Reply:
Actually I lied about the feature changes not breaking things, they might still be breaking for users that use `default-features = false` and `features = ["reqwest"]`, although it's not something incredibly hard to fix.

I added a compile error so that people who only have the `reqwest` feature enabled will be notified of the change instead of silently breaking references.